content/docs/hoon/guides/sail.md

```diff
@@ -0,0 +1,495 @@
++++
+title = "Sail (HTML)"
+weight = 6
+template = "doc.html"
++++
+
+Sail is a domain-specific language for composing HTML structures in Hoon.
```

Comment:
"HTML (and XML) structures"

Reply:
Oops okay, after refresh it's actually resolved, but I could still nitpick about getting the "xml means html" ordering right.